src/test/koa-esm-transform.test.ts

```diff
@@ -0,0 +1,332 @@
+/**
```

Comment:
Can you add a test for the case where the JS is invalid. I think we should probably serve the un-modified script?

Reply:
I totally needed to add error handling.  Thanks!  Added tests too.